src/ct/ct_treestore.cc

```diff
@@ -178,6 +178,11 @@ Glib::RefPtr<Gsv::Buffer> CtTreeIter::get_node_text_buffer() const
     return rRetTextBuffer;
 }
 
+bool CtTreeIter::get_node_buffer_is_style_applied() const
+{
+    return !!(*this)->get_value(_pColumns->rColTextBuffer);
```

Comment:
Is the double exclamation mark a typo?

Reply:
No, it's an old trick to convert something to bool. First ! make it negative, second one reverted it back. For some reason '(*this)->get_value(_pColumns->rColTextBuffer)' didn't want to be converted to bool, so I had to use the trick.

Reply:
Maybe I just need to compare the value with nullptr.